### PR TITLE
chore: Add hyperv generation to release notes, add dryrun option

### PR DESCRIPTION
**Pull request by bowang-666**
For VHD pipeline add hyperv generation to release notes, add dryrun option

### CODE REVIEW

File: .pipelines/.vsts-vhd-builder-release.yaml [hunk 1]
```diff
@@ -22,6 +22,10 @@ parameters:
   displayName: Build 1804 Gen2 GPU
   type: boolean
   default: true
+- name: nodryrun
```
Comment (xuto2) [inline .pipelines/.vsts-vhd-builder-release.yaml:25]: why not call it dry run

File: .pipelines/.vsts-vhd-builder-release.yaml [hunk 2]
```diff
@@ -324,6 +338,10 @@ phases:
           echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_DS2_v2'
           echo '##vso[task.setvariable variable=FEATURE_FLAGS]None'
         displayName: Setup Build Variables
+      - bash: |
+          echo '##vso[task.setvariable variable=DRY_RUN]True'
```
Comment (xuto2) [inline .pipelines/.vsts-vhd-builder-release.yaml:342]: is it possible to just set ${{ parameters.dryrun }} here?

Comment (bowang-666) [reply .pipelines/.vsts-vhd-builder-release.yaml]: Looks like it can work

File: .pipelines/.vsts-vhd-builder-release.yaml (hunk 1 above)
Comment (bowang-666) [reply .pipelines/.vsts-vhd-builder-release.yaml]: updated

**Review by xuto2 — APPROVED** [review]

**Pull request merged**